### ISSUE TITLE
Loss of inventory before teleportation to Limbo

### ISSUE THREAD

**Issue by xily** [DimensionalDevelopment/DimDoors#81]
_Observed on DimensionalDoors-3.0.9-287 w/ latest Forge modloader_

- When I die inside a dungeon dimension, I die for real (and drop all inventory) and get teleported to Limbo only after I click on Respawn button. 
- Also, monoliths in dungeon dimension don't seem to teleport me to Limbo.

Legacy behavior would have me teleported to Limbo instead of actually dying. 

Is this a new feature or an unfortunate bug?

**Comment by Runemoro** [comment]
Yes, the default behaviour was changed. We thought it would be better this way, since the loot in the dungeon is rare, so the risk should be high too. It also makes exploring dungeons more risky, so the traps actually scare the player this way.

**Comment by xily** [comment]
Thank you very much for the quick reply. I understand the choice. 
Is there any plan to make this a configurable option?

**Comment by Runemoro** [comment]
Yes, we'll add a config option.

**Comment by Runemoro** [comment]
Leave it open until we implement the config option. Otherwise we might forget.

**Comment by Darkmega18** [comment]
has this config been made yet? Cause I was unaware of this change to make dimension door related deaths kill your stuff and also you for spooky factor... 

I ask since someone playing my modpack, "Transcendence" came to me about losing their stuff in a dimension door pitfall and had the same idea of not losing stuff when being limbo'd and thought it was related to a tombmanygraves grave eating their stuff into a grave after death before being transported.

So this was actually your intended change to make it more hardcore and scary?

**Comment by Robijnvogel** [comment]
We haven't released any update since this issue was made, so no, it's not in the mod at the moment.

**Comment by Darkmega18** [comment]
Just asking. since theres that hardcore limbo config, but now that I think on it again, thats the thing where death in limbo respawns you there or something, isn't it? :V

**Comment by Baughn** [comment]
Any workarounds for this? It renders the mod effectively incompatible with TombManyGraves.

**Comment by Darkmega18** [comment]
I think tomb many will spawn your grave close to the void... the idea would be, to be very careful, and try to get back there and if you're vanilla, water scaffold your way down there and get your stuff if you're gutsy. or use any other shenanigans you can. 

Although, alternatively my friend told me one time after he got voided that corail tombs gravestones apparently the last point you were on solid ground. since in my last pack with corails, it was fine. he popped into the room that was destroyed to find his grave right there where he dropped off.

corail also has that, fast return thing. which is really damn nice. and the minor magical side of it's death stuff, and already comes pre-equipped with bauble and aether compat and stuff, which I think were a couple of the big issues with tombmany if you don't do addons for it. give it a shot. alternatively, some form of inventory saver like grave pet in inventory pets or soul bound or spectre anchors in random things might do the trick for key items. even some form of recall that works between dimensions to clutch yourself out. ;)
